portfolio.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Site</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+  </head>
+
+  <body>
+    <header class="navtools_container">
+      <h2 class="logo">Welcome</h2>
+ 
+      <nav class="navbar">
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+    </header>
+
+    <main class="main_container">
+      <h1 class="item1">Hey there. I'm Hamza</h1>
+
+      <h1 class="item2">I'm a software developer</h1>
+
+      <P class="item3">
+        I can help you build a product, feature or website 
+        Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.
+      </P>
+
+      <image class="item4" src="social media icons (2).png" alt="social media icons"></image>
```

Comment:
- [ ] Good job adding the icon image, however :smile:  you should not include icons in your web page as one image, but as independent clickable icons :+1: ,  as required you should use `ul` and `li` to perform that feature, kindly update that :smile_cat: , it will increase the user experience :+1: 

### You could use

```bash
<ul>
    <li>
         <a href="icon-addr">icon<a>
         ...
    </li>
</ul>